{{cookiecutter.github_repository}}/provisioner/roles/nginx/templates/site.80.conf.j2

```diff
@@ -10,7 +10,18 @@ server {
         }
     {% endif %}
 
+    {% if vm and (nginx_cert.stat.exists == false or nginx_key.stat.exists == false) %}
+      location / {
+            proxy_pass         http://127.0.0.1:8000;
```

Comment:
I think we need to use uwsgi_pass  and use the socket connection. 443 conf also uses sockets.